src/wolfio.c

```diff
@@ -770,7 +770,7 @@ int wolfIO_TcpConnect(SOCKET_T* sockfd, const char* ip, word16 port, int to_sec)
     int ret = 0;
     SOCKADDR_S addr;
     int sockaddr_len = sizeof(SOCKADDR_IN);
-#ifdef HAVE_GETADDRINFO
+#ifndef WOLF_C99
```

Comment:
This is not portable.  Before, autoconf detected whether getaddrinfo was available.  Now, an embedded user, who probably won't turn on WOLF_C99  is going to have code compiled that assumes getaddrinfo is available.